lua/lvim/plugin-loader.lua

```diff
@@ -93,34 +93,7 @@ function plugin_loader.load(configurations)
   vim.opt.runtimepath:remove(join_paths(plugins_dir, "*"))
 
   local status_ok = xpcall(function()
-    local opts = {
-      install = {
-        missing = true,
-        colorscheme = { lvim.colorscheme, "lunar", "habamax" },
-      },
-      ui = {
-        border = "rounded",
-      },
-      root = plugins_dir,
-      git = {
-        timeout = 120,
-      },
-      lockfile = join_paths(get_config_dir(), "lazy-lock.json"),
-      performance = {
-        rtp = {
-          reset = false,
-        },
-      },
-      defaults = {
-        lazy = false,
-        version = nil,
-      },
-      readme = {
-        root = join_paths(get_runtime_dir(), "lazy", "readme"),
-      },
-    }
-
-    lazy.setup(configurations, opts)
+    lazy.setup(configurations, lvim.lazy.opts)
```

Comment:
```suggestion
    table.insert(lvim.lazy.opts.install.colorscheme, 1, lvim.colorscheme)
    lazy.setup(configurations, lvim.lazy.opts)
```
we can add the user's colorscheme to the opts here